action/MultiStepSimulation.action

```diff
@@ -0,0 +1,11 @@
+# Assuming the simulation is paused, simulate a finite number of steps and return to paused state. 
+# The action returns feedback after each complete step.
+# For a service alternative, see StepSimulation, which often makes more sense when doing a single step (steps = 1).
+
+uint64 steps                        # Action goal: step through the simulation loop this many times.
+---
+bool success                        # Return true if stepping is supported and was successful.
+string status_message               # Additional comments or error status.
+                                    # Calling with simulation not paused will return failure and an appropriate message.
+---
+uint64 completed_steps              # Action feedback: number of completed steps in this action so far.
```

Comment:
```suggestion
uint64 completed_steps              # number of completed steps in this action so far.
uint64 remaining_steps              # number of steps remaining to be completed in this action.
```

This will give not just the absolute value of the progress but also the relative progress without needing to have made the request. Also removing the "Action Feedback" string from the content that's in the Action Feedback section.